subcommittee/coding-guidelines/meetings/2025-07-30/minutes.md

```diff
@@ -0,0 +1,96 @@
+9. Round table
+
+## **Check-in area**
+
+**Please add your name, and an emoji that describes your day.**
+
+* Alex Celeste ☕  
+* Fernando Jose ☕  
+* Achim Kriso 🐦  
+* Christof Petig 🚲  
+* Tiago Manczak
+
+**Notetaker:**
+
+* Fernando
+
+For tips on how we take notes in the Safety-Critical Rust Consortium, please see the [Meeting Notetaker Role](https://github.com/rustfoundation/safety-critical-rust-consortium/blob/main/docs/notetaker-role.md) doc.
+
+## **Housekeeping section**
+
+* Document space: [coding-guidelines](https://github.com/rustfoundation/safety-critical-rust-consortium/tree/main/subcommittee/coding-guidelines)  
+* Zulip: [safety-critical-consortium: Coding Guidelines](https://rust-lang.zulipchat.com/#narrow/channel/445688-safety-critical-consortium/topic/Coding.20Guidelines)  
+* [Early Alpha Milestone](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/milestone/1)
+
+## **Tasks**
+
+* Search for the \[todo\] markers
+
+## **Meeting Minutes**
+
+* Approval of previous minutes.  
+* No new people introduced.  
+* Infrastructure topic, more than one pair of examples per guideline. Mention the other infrastructure topic.  
+* CERT guidelines.  
+  * 32 and 35?  
+  * Taking a look at [the issue inspired by INT34-C](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/156). Call for comments.  
+    * It is asked if this is already covered by the checked arithmetic guideline. Encouraged to comment on the issue. A comment will be added \[todo\].  
+  * Taking a look at [the issue about categorization](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/152). Call for comments. No comments.  
+  * Taking a look at the checked arithmetic and division by zero pull requests. Discussion on the overlap, which comes from the CERT rules.  
+  * [Discussing the issue about unchecked](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/148)   
+    * Stemming from [the comment about unsafe](https://github.com/rustfoundation/safety-critical-rust-coding-guidelines/issues/148#issuecomment-3089428786): subset-oriented approach and unsafe. Different approaches are possible based on how to deal with unsafe (blocks).  
+      * Historical context about the use of unsafe added. 3rd party code in cybersecurity. Probably wanted to ban unsafe in that case. In functional safety, then everything is under control. Then, interesting categories relating to drivers and what the actual definition of unsafe is, Rust definition? All in all, avoiding unsafe is desired, but banning it totally is going too far. It would rather be desired to push it to the bottom of the code. A follow-up comment: there will be rules (guidelines) which are only sensible to apply in the context of unsafe.  
```

Comment:
```suggestion
      * Historical context about the use of unsafe added. 3rd party code in cybersecurity. Probably wanted to ban unsafe in that case. In functional safety, differently, "everything is under control". Then, there are interesting categories relating to drivers and what the actual definition of unsafe is; beyond the Rust definition? All in all, avoiding unsafe is desired, but banning it totally is going too far. It would rather be desired to push it to the bottom of the code. A follow-up comment: there will be rules (guidelines) which are only sensible to apply in the context of unsafe.  
```